partner_multi_relation/__manifest__.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2013-2017 Therp BV <http://therp.nl>
```

Comment:
@ddufresne  IMHO Please added a `website` as described in https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10

Reply:
@nikul-serpentcs done